src/Raven.Server/Web/System/OngoingTasksHandler.cs

```diff
@@ -305,6 +305,15 @@ private IEnumerable<OngoingTask> CollectExternalReplicationTasks(List<ExternalRe
             return taskInfo;
         }
 
+        [RavenAction("/databases/*/admin/backup-task/delay", "POST", AuthorizationStatus.Operator)]
+        public async Task DelayBackupTask()
+        {
+            var id = GetLongQueryString("taskId");
+            var delayInHrs = GetTimeSpanQueryString("duration");
```

Comment:
```suggestion
            var delay = GetTimeSpanQueryString("duration");
```

Reply:
Check that we pass a "positive" delay.
Should throw if we get `TimeSpan.MinValue` for example.

Reply:
Done.